src/extension.ts

```diff
@@ -153,6 +171,18 @@ export function activate(context: vscode.ExtensionContext) {
     }
   });
 
+  vscode.tasks.onDidEndTaskProcess((event) => {
```

Comment:
What's the difference between `onDidEndTaskProcess` and `onDidEndTask` ?

Reply:
`onDidEndTaskProcess` is triggered when the underlying process has ended. It allows to ensure everything is completely done, and we have the final exitcode to determine the task completion status.